src/Webby/Server.hs

```diff
@@ -126,11 +126,25 @@ requestBodyLength = do
 finish :: WebbyM appEnv a
 finish = E.throwIO FinishThrown
 
+-- | Send an image in the response body. Also
+-- sets @Content-Type@ header to @image/{imgType}
+-- e.g. image/svg+xml
+image :: ByteString -> ByteString -> WebbyM appEnv ()
```

Comment:
`image` takes two `ByteString`s and this is a place for an easy mistake. I think we should take advantage of [mime-types](https://hackage.haskell.org/package/mime-types-0.1.1.0/docs/Network-Mime.html) package to,

1. Pick the right mime-type based on filename. e.g `thumbnail.jpg` will return `"image/jpeg"`, a `MimeType` value.
This applies to application code.

2. Use `MimeType` to indicate the content-type header to be set in `image`.
We can use this type to define `image`, e.g `image :: ByteString -> MimeType -> WebbyM appEnv ()`.

We can also export `raw` for advanced users who wish to set content-type separately.

Reply:
Thanks @krisis , I've incorporated your suggestions :+1: